pytensor/link/pytorch/dispatch/elemwise.py

```diff
@@ -160,3 +166,38 @@ def softmax_grad(dy, sm):
         return dy_times_sm - torch.sum(dy_times_sm, dim=axis, keepdim=True) * sm
 
     return softmax_grad
+
+
+def elemwise_scalar_loop(base_fn, op, node, **kwargs):
```

Comment:
This will work for any scalar function `base_fn` not just scalar loop, so may be worth making the docstrings a bit more generic?

Reply:
Yup,  good callout.